website/docs/plugin/framework/handling-data/blocks/list-nested.mdx

```diff
@@ -67,9 +68,8 @@ func (r ThingResource) Schema(ctx context.Context, req resource.SchemaRequest, r
                         },
                     },
                 },
-                // ... potentially other fields ...
```

Comment:
This should probably remain, as it's possible to set other fields at this level (e.g., `CustomType`, `Description` etc)